kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/ColumnMappingSuite.scala

```diff
@@ -201,6 +287,262 @@ class ColumnMappingSuite extends AnyFunSuite {
         (k: AnyRef) => assertThat(k).asString.startsWith("col-"))
   }
 
+  test("assign id and physical name to new table with nested struct type" +
+    " and enableIcebergCompatV2=true") {
+    val schema: StructType =
+      new StructType()
+        .add("a", StringType.STRING)
+        .add("b",
+          new StructType()
+            .add("d", IntegerType.INTEGER)
+            .add("e", IntegerType.INTEGER))
+        .add("c", IntegerType.INTEGER)
+
+    val metadata: Metadata =
+      ColumnMapping.updateColumnMappingMetadata(
+        createMetadata(schema).withNewConfiguration(
+          Maps.newHashMap(TableConfig.ICEBERG_COMPAT_V2_ENABLED.getKey, "true")), "id", true)
```

Comment:
can you please move `id and true` to a new line? took me a minute to figure out they are params to `updateColumnMappingMetadata`, and if you named the `true` param that would help with readability, too